toolkit/Makefile

```diff
@@ -132,14 +132,30 @@ PACKAGE_URL_LIST   += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR
 REPO_LIST          ?=
 SRPM_URL_LIST      ?= https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/base/srpms
 
+######## COMMON MAKEFILE UTILITIES ########
+
+# Misc function defines
+# Variable prerequisite tracking
+include $(SCRIPTS_DIR)/utils.mk
+
+######## REMAINING BUILD FLAGS ########
+
 ifneq ($(DAILY_BUILD_ID),)
    $(warning Using Daily Build $(DAILY_BUILD_ID))
+   # Ensure build_arch is set
+   ifeq ($(build_arch),)
+      $(error build_arch must be set when using DAILY_BUILD_ID)
+   endif
    # build_arch cannot be directly used because Azure storage do not support container names with '_' char
    ifeq ($(build_arch),x86_64)
-      override PACKAGE_URL_LIST   := https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILY_BUILD_ID)-amd64/built_rpms_all
+      # The actual repo is found at <URL>/, while a duplicate copy of all the rpms can be found at <URL>/built_rpms_all/
+      # Include both so that the tools that expect a valid repo work, while the tools that expect a basic URL also work.
+      override PACKAGE_URL_LIST   := https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILY_BUILD_ID)-amd64/built_rpms_all \
+                                       https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILY_BUILD_ID)-amd64
       override SRPM_URL_LIST      := https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILY_BUILD_ID)-amd64/SRPMS
    else
-      override PACKAGE_URL_LIST   := https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILY_BUILD_ID)-arm64/built_rpms_all
+      override PACKAGE_URL_LIST   := https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILY_BUILD_ID)-arm64/built_rpms_all \
+                                       https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILY_BUILD_ID)-arm64/
```

Comment:
there is a slash at the end of the arm64 url here, but not amd64. they should be consistent. i think it’s actually not needed

Reply:
```suggestion
                                       https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILY_BUILD_ID)-arm64
```